docs/Community/sig-ci/README.md

```diff
@@ -0,0 +1,34 @@
+Continuous Integration/Continuous Deployment (CI/CD) Special Interest Group
+==========================================================================
+
+This SIG is responsible for infrastructure and automation related to CI/CD for the Open Enclave SDK project.
+
+This SIG may also weigh in on specific PRs to ensure that it is possible to build CI/CD to automate testing the changes suggested in the PR.
```

Comment:
This is probably up to the SIG chair to amend as needed, but I imagine that for the CD part of this SIG, they will also likely have input related to matters of release and platform targeting.

Reply:
Added text that effect.